lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,200 @@
+= Exploring Exponential Models
+
+@description{Students investigate exponential relationships in data about Covid spread, using an inquiry-based model of sense-making, hypothesizing, modeling and testing. They are introduced to table transformations and inverse functions, which are used to fit exponential models onto nonlinear data.}
+
+@lesson-prereqs{exploring-quadratic-models}
+
+@keywords{exponential, quadratic, square, coefficient}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Read and interpret real-world data, presented in a scatter plot
+* Model exponential relationships using functions
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Pyret to model exponential relationships in data
+* Let's use Pyret to filter a dataset into subsets
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* All students should pull up their @online-exercise{program-list} and, open the @starter-file{alg2-covid}.
+
+| Key Points For The Facilitator
+|
+* TBD
+|===
+
+== Filtering {amp} Simpson's Paradox @duration{20 minutes}
+
+=== Overview
+Students explore the @starter-file{alg2-covid}, and discover that some scatter plots show __multiple__ trends instead of just one! This motivates a brief discussion of Simpson's Paradox, and teaches them to filter their dataset into grouped samples.
+
+=== Launch
+
+[.lesson-instruction]
+- Open the @starter-file{alg2-covid}
+- From the File menu, select "Save a Copy", and click "Run"
+- Working in pairs or small groups, complete questions 1-10 of @printable-exercise{exploring-covid.adoc}
+
+@center{@image{images/multiple-models.png}}
+
+The scatterplot we just produced doesn't look much like a scatterplot all! It looks like someone took a blue marker and drew in five different curvy lines. While it's clear that there are strong patterns in the data, these patterns are so distinct from one another that there isn't really one, single model that fits them all. Each relationship appears very strong, almost as if there is _more than one model_ here.
+
+Review student answers to confirm that students have made a number of observations:
+
+* There is _more than one_ relationship in this dataset
+* Every relationship seems extremely strong
+* Most/all relationships appear non-linear
+
+=== Investigate
+
+With all these clear, tight lines, we might think this would be a dataset with a very strong correlation. Unfortunately, that's not true at all.
+
```

Comment:
Why tell them this here? Wouldn't you want them to discover this before explaining it? I'm not curious yet.

Reply:
Good call. Added discuss questions just before this section.